file-agent-kafka-client/src/main/java/im/ligas/kafka/stream/MetaData.java

```diff
@@ -0,0 +1,88 @@
+package im.ligas.kafka.stream;
+
+public class MetaData {
```

Comment:
this needs to be extracted into a separate jar lib with all the other DTO objects